lib/agent/https_proxy_ocsp_agent.js

```diff
@@ -1,278 +1,33 @@
 /*
- * Copyright (c) 2013 Nathan Rajlich <nathan@tootallnate.net>
+ * Copyright (c) 2015-2023 Snowflake Computing Inc. All rights reserved.
  */
```

Comment:
The GitHub diff looks horrible, but basically:

I have removed the old implementation that re-implemented some of the `https-proxy-agent` code, and this module is now a thin wrapper over `https-proxy-agent`.

As such, I updated the license header to reflect that this file now contains original code.